lib/charms/opensearch/v0/opensearch_backups.py

```diff
@@ -389,10 +493,15 @@ def _list_backups(self) -> Dict[int, str]:
         }
 
     def is_backup_in_progress(self, backup_id=None) -> bool:
-        """Returns True if backup is in progress, False otherwise."""
-        return self._query_backup_status(backup_id) in [
-            BackupServiceState.SNAPSHOT_IN_PROGRESS,
-        ]
+        """Returns True if backup is in progress, False otherwise.
+
+        We filter the _query_backup_status() and seek for the following states:
+        - SNAPSHOT_IN_PROGRESS
+        """
+        if self._query_backup_status() in [BackupServiceState.SNAPSHOT_IN_PROGRESS]:
```

Comment:
This is a proxy method, we should perhaps rename this method `_backup_status` and put the code from `_query_backup_status` here and test on the status directly instead of a boolean.